modules/ar/include/visp3/ar/vpPanda3DPostProcessFilter.h

```diff
@@ -62,13 +62,27 @@ class VISP_EXPORT vpPanda3DPostProcessFilter : public vpPanda3DBaseRenderer
     m_renderOrder = m_inputRenderer->getRenderOrder() + 1;
   }
 
+  ~vpPanda3DPostProcessFilter()
+  {
+    // delete m_texture;
+    // delete m_buffer;
+  }
```

Comment:
I will remove them.
I was under the impression that Panda3D managed the texture and graphics buffer (through the WindowFramework). after having another look at the examples, I'm not so sure. I will try to replace the raw pointers with Panda's managed memory system (using `PointerTo`), so that there are no clashes with panda potentially releasing the memory by itself.